### PR TITLE
Update project readme

### PR DESCRIPTION
**Pull request by folksgl**
Update readme info to reflect framework switch 

Unpin black language version from 3.9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,42 +44,17 @@ python -m pip install -r requirements-dev.txt
...
 ### Running the application
-After completing [development setup](#development-setup) and
-[environment variable setup](#required-environment-variables) you can run the
-application locally with:
+After completing [development setup](#development-setup) application locally with:
 ```shell
-python manage.py collectstatic
-python manage.py test --debug-mode
-gunicorn -b 127.0.0.1:8080 usps.wsgi
+pytest # Unit tests should be passing
```
Comment (shumatepf) [inline README.md:50]: This should be
```
pytest usps/tests.py
```

File: README.md [hunk 2]
```diff
@@ -43,42 +44,17 @@ python -m pip install -r requirements-dev.txt
...
+uvicorn usps.main:app
 ```
 
+### Viewing API Endpoints and documentation
+Documentation can be viewed locally by running the application and visiting
+http://127.0.0.1:8000/docs
```
Comment (shumatepf) [inline README.md:56]: This points to the swagger doc. Do we want to show off the redoc instead? That would be http://127.0.0.1:8000/redoc

File: README.md (hunk 1 above)
Comment (shumatepf) [reply README.md:50]: It should also be mentioned that the tests will fail if `DEBUG` is not set to `TRUE`.

File: README.md (hunk 2 above)
Comment (folksgl) [reply README.md:56]: Yes. I agree the redoc docs look a bit nicer.

File: README.md (hunk 1 above)
Comment (folksgl) [reply README.md:50]: Actually, the command should be `python -m pytest` and we should follow the standard pytest structure of having a `tests/` directory in the root of our project. That way pytest test discovery rules will automatically find our tests. Making these changes.

**Review by shumatepf — APPROVED** [review]

**Review by shumatepf — APPROVED** [review]

**Pull request merged**